samples/Rakefile

```diff
@@ -120,7 +128,8 @@ task :run => :download_jars do |t|
   end
   log_configuration = ENV['log_configuration']
   puts "Running the Kinesis processing application..."
-  classpath = FileList["#{JAR_DIR}/*.jar"].join(':')
+
+  classpath = MAVEN_PACKAGES.map { |jar| get_maven_jar_info(*jar).last }.sort.join(':')
```

Comment:
This change will allow us to install the new .jar files alongside the old ones, but still only use the new ones when the KCL is run.